core/deps/CMakeLists.txt

```
@@ -48,14 +50,21 @@ target_include_directories(miniz
 if(NOT TANGRAM_USE_SYSTEM_FONT_LIBS)
   ## Harfbuzz - ICU-Common - Freetype2 ##
   #######################################
-  set(HARFBUZZ_BUILD_ICU ON CACHE BOOL "Enable building of ICU")
-  add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/harfbuzz-icu-freetype)
-
-  message(STATUS "harfbuzz" ${HARFBUZZ_LIBRARIES})
+  if(WIN32)
```

This is problematic for scenarios where you don't want to use in-system packages (e.g. no vcpkg, libs installed in program files).

- Assuming I'm building using MinGW, I want it to use the submodule instead of find_package
- Assuming you're building with MSVC, you want the current apprach, I guess (are you getting compile errors when building them too?)

```suggestion
  if(MSVC)
```

---

Done